.github/workflows/ray.yml

```
@@ -0,0 +1,35 @@
+name: ray
+
+on:
+  schedule:
+    - cron: '0 15 * * *'
+  pull_request:
+    paths:
+      - 'ray/**.py'
```

Not really necessary but for consistency.
```suggestion
      - 'ray/**'
```